src/java/picard/analysis/CollectInsertSizeMetrics.java

```diff
@@ -48,13 +48,39 @@
  * @author Doug Voet (dvoet at broadinstitute dot org)
  */
 @CommandLineProgramProperties(
-        usage = "Reads a SAM or BAM file and writes a file containing metrics about " +
-                "the statistical distribution of insert size (excluding duplicates) " +
-                "and generates a Histogram plot.",
-        usageShort = "Writes insert size distribution metrics for a SAM or BAM file",
+        usage = CollectInsertSizeMetrics.USAGE_SUMMARY,
+        usageShort = CollectInsertSizeMetrics.USAGE_BRIEF,
         programGroup = Metrics.class
 )
 public class CollectInsertSizeMetrics extends SinglePassSamProgram {
+    static final String USAGE_BRIEF = "Metrics about the insert size distribution of a paired-end library";
+    static final String USAGE_SUMMARY = "Metrics about the insert size distribution of a paired-end library, created by the " +
+            "CollectInsertSizeMetrics program and usually written to a file with the extension " +
```

Comment:
This is a weird phrasing, explaining the tool by defining its output. I would prefer we keep the same pattern as previously ("This tool does X") throughout the docs.